cli/determined_cli/experiment.py

```diff
@@ -466,34 +410,33 @@ def describe(args: Namespace) -> None:
     )
 
     values = []
-    for e in experiments:
-        for t in e.trials:
-            for step in t.steps:
+    for doc in docs:
+        for trial in doc["trials"]:
+            for step in trial["steps"]:
                 t_metrics_fields = []
-                if hasattr(step, "metrics"):
-                    avg_metrics = step.metrics
+                if step.get("metrics"):
+                    avg_metrics = step["metrics"]["avg_metrics"]
                     for name in t_metrics_names:
                         if name in avg_metrics:
                             t_metrics_fields.append(avg_metrics[name])
                         else:
                             t_metrics_fields.append(None)
 
-                checkpoint = step.checkpoint
+                checkpoint = step.get("checkpoint")
                 if checkpoint:
-                    checkpoint_state = checkpoint.state
-                    checkpoint_start_time = checkpoint.start_time
-                    checkpoint_end_time = checkpoint.end_time
+                    checkpoint_state = checkpoint["state"]
```

Comment:
We could maybe condense this section by setting checkpoint to `{}` if it doesn't exist and then using `checkpoint.get("field", None)` to set the following variables